src/main/java/uk/gov/pay/webhooks/app/WebhooksModule.java

```diff
@@ -58,8 +60,14 @@ public IdGenerator externalIdGenerator() {
 
     @Provides
     @Singleton
-    public java.net.http.HttpClient netHttpClient() {
-        return java.net.http.HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(5)).build();
+    public Client netHttpClient() {
+        ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+        Client client = clientBuilder.build();
```

Comment:
In all the other apps, we set the `SSLContext` on the builder. Is there a reason this isn't done here?
Sounds like it might be needed to make HTTPS requests but haven't read too much into it.